lib/libzutil/zutil_import.c

```diff
@@ -929,22 +1006,43 @@ zpool_read_label(int fd, nvlist_t **config, int *num_labels)
 
 	if (lio_listio(LIO_WAIT, aiocbps, VDEV_LABELS, NULL) != 0) {
 		int saved_errno = errno;
+		boolean_t do_slow = B_FALSE;
+		error = -1;
 
 		if (errno == EAGAIN || errno == EINTR || errno == EIO) {
 			/*
-			 * A portion of the requests may have been submitted.
-			 * Clean them up.
+			 * A portion of the requests may have
+			 * been submitted. Clean them up.
```

Comment:
Having moved this comment back to where it started, it now looks like you unnecessarily wrapped it.

Reply:
Sounds like something I'd do, moving things around the long way only to wind up in the identical place.

Done.